src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/CompiledRegexRunner.cs

```diff
@@ -1,27 +1,32 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Buffers;
 using System.Globalization;
 
 namespace System.Text.RegularExpressions
 {
     internal sealed class CompiledRegexRunner : RegexRunner
     {
         private readonly ScanDelegate _scanMethod;
-        /// <summary>This field will only be set if the pattern contains backreferences and has RegexOptions.IgnoreCase</summary>
-        private readonly CultureInfo? _culture;
 
 #pragma warning disable CA1823 // Avoid unused private fields. Justification: Used via reflection to cache the Case behavior if needed.
 #pragma warning disable CS0169
+        private readonly IndexOfAnyValues<char>[]? _indexOfAnyValues;
+
+        /// <summary>This field will only be set if the pattern contains backreferences and has RegexOptions.IgnoreCase</summary>
+        private readonly CultureInfo? _culture;
```

Comment:
Moving this is fine, but I'm curious, is this just style or did something cause there to start being a warning on that existing field as well?

Reply:
I think I just expected that a similar warning would fire for the new field, but I see now that `_caseBehavior` is the only one not set by the ctor ... I don't remember why I moved the culture as well.

I suppose neither `_indexOfAnyValues` nor `_culture` actually need to be in the pragma.